Feature/pagination filtration sorting

AndrianovaVeronika opened the pull request:
Sorting and pagination features

Commit:
feat: ServerPaginationGrid
fix: styles, use ServerPaginationGrid instead of DataGrid

Commit:
feat: ServerPaginationGrid sorting ability

nik-clockwise commented on src/components/DataTables/OrdersTable/index.js:
```diff
@@ -55,19 +55,15 @@ const OrdersTable = () => {
-            filtrate={filtrate}
+            setFilters={setFilters}
         />
         <DataTable
```
Тут не нужен  ServerPaginationGrid?

AndrianovaVeronika replied:
поменяла

nik-clockwise commented on src/components/DataTables/UsersTable/index.js:
```diff
@@ -40,25 +39,22 @@ const UsersTable = () => {
-                filtrate={filtrate}
+                setFilters={setFilters}
             />
             <DataTable
```
Тоже не нужен  ServerPaginationGrid?

AndrianovaVeronika replied:
+

Commit:
fix: rewrite to ServerPaginationGrid, rewrite the way of retriewing values, ServerPaginationGrid, the way of retrieving available masters

Commit:
"October 21 weekly remarks"
feat: text Input on PriceRange
fix: styles, MasterOrders filter through login (name or email)
